master/security/service-accounts.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Use service accounts in policy rules
+---
+
+### Big picture
+
+Use Calico network policy to allow/deny traffic for Kubernetes services. 
+
+### Value
+
+Using Calico network policy, you can leverage Kubernetes service accounts with RBAC for flexible control over how policies are applied in a cluster. For example, the network security can have RBAC permissions to:
+
+- Control which service accounts the developer team can use within a namespace
+- Write high-priority network policies for those service accounts (that the developer team cannot override) 
+
+The network security team can maintain full control of security, while selectively allowing developer operations where it makes sense.  
+
+Using **Istio-enabled apps** with Calico network policy, the cryptographic identity associated with the service account is checked (along with the network identity) to achieve two-factor authentication.
+
+### Features
+
+This how-to guide uses the following Calico features:
+
+**NetworkPolicy** or **GlobalNetworkPolicy** with a service account rule and match criteria.
+
+
+### Concepts
+
+#### Use smallest set of permissions required
+
+Service accounts are controlled by RBAC so you can grant permissions to trusted entities (code and/or people). To perform any operation in a workload, clients are required to authenticate with the Kubernetes API server. If you do not explicitly assign a service account to a pod, it uses the default ServiceAccount in the namespace. You should not grant broad permissions to the default service account for a namespace. If an application needs access to the Kubernetes API, create separate service accounts with the smallest set of permissions required. 
```

Comment:
Said this in a few PRs now - this line is too long. 

Notice how the GitHub UI treats this all as line 31. This should probably be no fewer than 4 distinct lines with explicit line returns in between. It makes it harder to use the GitHub UI to review when the lines are long, and also displays poorly in many common editors that won't add automatic breaks.

Reply:
Fixed. I have created separate lines, but this goes against all writing standards to break up sentences based on editor functions.